src/components/Inbox/index.tsx

```diff
@@ -1,88 +1,185 @@
-        aria-label="unresolved"
-        subheader={(
-          <ListSubheader classes={{ root: 'section-header' }} disableGutters>
-            <IconButton
-              classes={{ root: 'section-header-button' }}
-              aria-label="dropdown"
-              disableRipple
-            >
-              <ArrowDropDownIcon
-                classes={{ root: 'section-header-button-icon' }}
-              />
-            </IconButton>
-            Unresolved
-          </ListSubheader>
-        )}
-      >
-        <Divider classes={{ root: 'divider' }} />
-        <ListItem button classes={{ root: 'list-item' }}>
-          <ListItemIcon classes={{ root: 'icon' }}>
-            <Checkbox
-              classes={{
-                root: 'checkbox',
-                checked: 'checkbox-checked',
-                colorSecondary: 'checkbox-checked',
-              }}
-              size="small"
-              disableTouchRipple
-              color="default"
-            />
-          </ListItemIcon>
-          <ListItemText
-            classes={{
-              root: 'text',
-              primary: 'primary-text',
-              secondary: 'secondary-text',
-            }}
-            primary="Button not working"
-            secondary="The popular times button does nothing"
-          />
-        </ListItem>
-        <ListItem button classes={{ root: 'list-item', selected: 'list-item-focused' }}>
-          <ListItemIcon classes={{ root: 'icon' }}>
-            <Checkbox
-              classes={{
-                root: 'checkbox',
-                checked: 'checkbox-checked',
-                colorSecondary: 'checkbox-checked',
-              }}
-              size="small"
-              disableTouchRipple
-              color="default"
-            />
-          </ListItemIcon>
-          <ListItemText
-            classes={{
-              root: 'text',
-              primary: 'primary-text',
-              secondary: 'secondary-text',
-            }}
-            primary="Really really really really really really reaalllllly long"
-            secondary="asdfaskjfdhflkajghajwehrafbjdknsfhqguefhabjdknhluky2gevbf"
-          />
-        </ListItem>
-      </List>
-    </div>
-  </StylesProvider>
-);
+interface InboxProps extends DispatchProps {
+  unresolvedItems: InboxItem[]
+  resolvedItems: InboxItem[]
+}
 
-export default Inbox;
+interface InboxState {
+  unresolvedOpen: boolean,
+  resolvedOpen: boolean,
+}
+
+enum InboxSection {
+  Unresolved = 'Unresolved',
+  Resolved = 'Resolved',
+}
+
+class Inbox extends React.Component<InboxProps, InboxState> {
+  constructor(props: any) {
+    super(props);
+    this.state = {
+      unresolvedOpen: true,
+      resolvedOpen: true,
+    };
+  }
+
+  handleClick = (sectionName: string) => {
+    if (sectionName === InboxSection.Unresolved) {
+      this.setState((state) => ({
+        unresolvedOpen: !state.unresolvedOpen,
+        resolvedOpen: state.resolvedOpen,
+      }));
```

Comment:
not a big deal since its only one line but you can do:
```
{
...state,
unresolvedOpen: !state.unresolvedOpen
}
```
since you're only changing one field